cppclient/tests/api_json.cc

```diff
@@ -62,5 +62,38 @@ TEST_F(IotaJsonAPITest, GetBalances) {
   EXPECT_EQ(api.request, req);
   EXPECT_EQ(response, expected);
 }
+TEST_F(IotaJsonAPITest, GetNodeInfo) {
+  MockAPI api;
+
+  json req;
+  req["command"] = "getNodeInfo";
+  json res;
+  char *str = "VBVEUQYE99LFWHDZRFKTGFHYGDFEAMAEBGUBTTJRFKHCFBRTXFAJQ9XIUEZQCJOQTZNOOHKUQIKOY9999";
```

Comment:
Avoid C-style string. Instead, make use of C++ `std::string`, which provides abstractions along with better memory management.